src/components/ExperienceDetail/ReportModal/ReportForm/ReportForm.js

```diff
@@ -21,15 +21,21 @@ import {
 } from './constants';
 
 const getReasonCategoryOptions = reportType => {
-  return reportType === REPORT_TYPE.SALARY
-    ? salaryReportReasons
-    : experienceReportReasons;
+  switch (reportType) {
+    case REPORT_TYPE.SALARY:
+      return salaryReportReasons;
+    default:
```

Comment:
忘記說，用Switch 的好處是就可以把所有 case 都寫出來
```suggestion
    case REPORT_TYPE.EXPERIENCE:
```

L36也是~

Reply:
我一開始也是猶豫是否要全部都寫出來，因為只有兩項，default 會重複，後來想到 default 給 []，好像 [] 比較易讀一點？

- default 重複
```javascript
const getReasonCategoryOptions = reportType => {
  switch (reportType) {
    case REPORT_TYPE.SALARY:
      return salaryReportReasons;
    case REPORT_TYPE.EXPERIENCE:
      return experienceReportReasons;
    default:
      return experienceReportReasons;
  }
};
```
- default 給 []
```javascript
const getReasonCategoryOptions = reportType => {
  switch (reportType) {
    case REPORT_TYPE.SALARY:
      return salaryReportReasons;
    case REPORT_TYPE.EXPERIENCE:
      return experienceReportReasons;
    default:
      return [];
  }
};
```